classes/genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_reader :items, :name
+
+  def initialize(name, items)
+    @id = 0
+    @name = name
+    @items = items
+  end
+
+  def add_item(item)
+    @items << item
+    item.add_genre = self
+  end
+end
```

Comment:
[OPTIONAL] I just wanted to ask where are we pushing the items to? isn't `@items` is supposed to be an array.
basically where are we adding the items with the add_item method. 
Thanks

Reply:
@PinkMoon25 Thank you for your question
You are right `@item` is an array reason why you can see the use of ` @item << item `in this function `add_item`
If you look well at the constructor method I assumed that sometimes you may want to create a genre instance with some items inside You can do that by passing an array of items as the constructor's second argument.
For instance, let's have look at this example which illustrates what I am saying
`Genre.new("name",[item,item])`